### PR TITLE
add NODE_MODULE_VERSION column to the releases table

### PR DESCRIPTION
**Pull request by max-mapper**
why this is valuable: https://gist.github.com/maxogden/e81e0f1d222106146828

what it looks like:

screen shot 2015-09-15 at 2 51 21 pm

### CODE REVIEW

**Comment by Fishrock123** [comment]
@maxogden Correct me if I'm wrong, but isn't this better covered by https://nodejs.org/dist/index.json?

**Comment by Fishrock123** [comment]
Perhaps we should link to that json on this page?

**Comment by max-mapper** [comment]
@Fishrock123 the data you linked is that data that is used to render the table, so it's the same thing

**Comment by rvagg** [comment]
@maxogden see also https://github.com/nodejs/build/issues/151 which is related to your gist

**Comment by rvagg** [comment]
lgtm, I quite like the idea of this number becoming part of user-base consciousness as it's pretty important

**Comment by fhemberger** [comment]
Can someone please add a little note on the table, what "NODE_MODULE_VERSION" is supposed to mean? Have you checked the table layout on small screens as well?

**Comment by rvagg** [comment]
we could just call it "modules version", that'd be consistent enough:

```
$ node -p process.versions
{ http_parser: '2.5.0',
  node: '4.1.0',
  v8: '4.5.103.33',
  uv: '1.7.4',
  zlib: '1.2.8',
  ares: '1.10.1-DEV',
  modules: '46',                         <————
  openssl: '1.0.2d' }
```

**Comment by rvagg** [comment]
Or just "Modules", it seems confusing but with a footnote it might make sense:

> ‡ "Modules" refers to the ABI (application binary interface) version number of Node.js, used to determine which versions of Node.js compiled C++ add-on binaries can be loaded in to without needing to be re-compiled.

**Comment by fhemberger** [comment]
@rvagg :+1:

**Comment by Fishrock123** [comment]
Rod's comment seems good.

**Comment by max-mapper** [comment]
Updated:

screen shot 2015-09-19 at 9 47 26 am

File: locale/en/download/releases.md [hunk 1]
```diff
@@ -3,4 +3,5 @@ layout: download-releases.hbs
 title: Previous Releases
 iojs:
   intro: "Releases 1.x through 3.x were called \"io.js\" as they were part of the io.js fork. As of Node.js 4.0.0 the former release lines of io.js converged with Node.js 0.12.x into unified Node.js releases."
+modules: "\"Modules\" refers to the ABI (application binary interface) version number of Node.js, used to determine which versions of Node.js compiled C++ add-on binaries can be loaded in to without needing to be re-compiled. This version number is also referred to as NODE_MODULE_VERSION."
```
Comment (Fishrock123) [inline locale/en/download/releases.md]: > `NODE_MODULE_VERSION`.

Might render better. (Also: period.) :)

Comment (max-mapper) [inline locale/en/download/releases.md]: @Fishrock123 the period is in the diff, not in the screenshot sorry. Heres what it looks like wrapped with a `<code>`:

screen shot 2015-09-19 at 10 09 59 am

And without (the current version in the branch now):

screen shot 2015-09-19 at 10 10 29 am

Comment (fhemberger) [inline locale/en/download/releases.md]: Looks better with `<code>` IMHO. Let's get this merged. ;)

**Comment by max-mapper** [comment]
alright, updated. should be good to go

**Comment by fhemberger** [comment]
Yaaay! Thank you! ^____^

**Comment by rvagg** [comment]
great work! I love that this is first-class, thanks @maxogden

**Pull request merged**